spring-cloud/spring-cloud-openfeign/src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+spring.application.name= openfeign
+logging.level.com.baeldung.cloud.openfeign.client.JSONPlaceHolderClient: DEBUG
```

Comment:
Reading the article, I was curious about whether you had to set logging level on a per-interface basis, or whether something like:

`logging.level.com.baeldung.cloud.openfeign.client: DEBUG`

Would set a whole package's debug level. If it can, then perhaps that's a more useful example?